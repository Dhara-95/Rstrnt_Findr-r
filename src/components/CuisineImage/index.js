import React from "react";

const CuisineImage = (props) => {
  const { image } = props;

  return (
    <div className="w-60 h-60 bg-grey rounded-3xl text-black m-4">{image}</div>
  );
};

export { CuisineImage };
