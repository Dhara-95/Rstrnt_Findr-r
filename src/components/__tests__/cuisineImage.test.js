import React from "react";
import renderer from "react-test-renderer";

import { CuisineImage } from "../CuisineImage";

it("displays the cuisine images correctly", () => {
  const tree = renderer.create(<CuisineImage image={"image"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
