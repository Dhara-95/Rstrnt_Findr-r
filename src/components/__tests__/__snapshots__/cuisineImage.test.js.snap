// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the cuisine images correctly 1`] = `
<div
  className="w-60 h-60 bg-grey rounded-3xl text-black m-4"
>
  image
</div>
`;
